INSTALL.md

```diff
@@ -103,8 +108,257 @@ docker container stop elasticsearch_test; docker container rm elasticsearch_test
+
+| **MAIN** | **Result** |
+|:---:|:--- |
+| `main` | Standard OPTIMADE server. |
+| `main_index` | Index meta-database OPTIMADE server. |
+
+Using Docker, the following command will run a container from the image:
+
+```shell
+# rm will remove container when it exits.
+# detach will run the server in the background.
+# publish will run the server from the host port 8080.
+# name will give the container a handy name for referencing later.
+docker run \
+    --rm \
+    --detach \
+    --publish 8080:5000 \
+    --env MAIN=main \
+    --name my-optimade \
+    ghcr.io/materials-consortia/optimade:latest
+```
+
+The server should now be available at [localhost:8080](http://localhost:8080).
+
+### Configure the server
+
+The server will use test data and a test configuration by default.
+This can be changed in several different ways.
+
+One is to `git clone` the repository locally and bind the repository folder to the `/app` folder in the container:
+
+```shell
+# volume will bind the local version of `optimade-python-tools` to the container.
+docker run \
+    --rm \
+    --detach \
+    --publish 8080:5000 \
+    --env MAIN=main \
+    --name my-optimade \
+    --volume /path/to/optimade-python-tools:/app \
+    ghcr.io/materials-consortia/optimade:latest
+```
+
+Where you should change the `/path/to/optimade-python-tools` to the full local path to the repository - or use `$PWD` if you are running this command from the root of the cloned repository on a Unix system.
+Equivalently, `%cd%` should work on Windows.
+
+In this setup you can change the repository root file `optimade_config.json` with the appropriate information.
+E.g., if you do not wish to use the test data you can add the `insert_test_data` key with a value of `false`.
+
+Another option is to _not_ `git clone` the repository locally, but instead only generate a JSON or YAML file somewhere, where the directory of its location is bound to another location in the container that is not used, e.g., `/config` or similar.
+As an example, let us say we generate `config.yml` locally, with the full path `/home/user/optimade/config.yml`.
+Then we can start the server with the following command:
+
+```shell
+# volume will bind the local directory to an unused folder in the container.
+docker run \
+    --rm \
+    --detach \
+    --publish 8080:5000 \
+    --env MAIN=main \
+    --name my-optimade \
+    --volume /home/user/optimade:/config \
+    --env OPTIMADE_CONFIG_FILE=/config/config.yml \
+    ghcr.io/materials-consortia/optimade:latest
+```
+
+As shown, it is necessary to update the environment variable `OPTIMADE_CONFIG_FILE` within the container to point to the new internal path to the config file.
+By default, this environment variable points to `/app/optimade_config.json`.
+
+This also reveals another way of configuring the server: Use environment variables when running the container that supplies the values that would otherwise be supplied from the configuration file.
```

Comment:
```suggestion
This also reveals another way of configuring the server: set environment variables when running the container to supply values that would otherwise be supplied from the configuration file.
```